codex.nim

```diff
@@ -54,91 +55,93 @@ when isMainModule:
   config.setupLogging()
   config.setupMetrics()
 
-  case config.cmd:
-  of StartUpCommand.noCommand:
+  if config.nat == ValidIpAddress.init(IPv4_any()):
+    error "`--nat` cannot be set to the any (`0.0.0.0`) address"
+    quit QuitFailure
 
-    if config.nat == ValidIpAddress.init(IPv4_any()):
-      error "`--nat` cannot be set to the any (`0.0.0.0`) address"
-      quit QuitFailure
+  if config.nat == ValidIpAddress.init("127.0.0.1"):
+    warn "`--nat` is set to loopback, your node wont properly announce over the DHT"
 
-    if config.nat == ValidIpAddress.init("127.0.0.1"):
-      warn "`--nat` is set to loopback, your node wont properly announce over the DHT"
+  if not(checkAndCreateDataDir((config.dataDir).string)):
+    # We are unable to access/create data folder or data folder's
+    # permissions are insecure.
+    quit QuitFailure
 
-    if not(checkAndCreateDataDir((config.dataDir).string)):
-      # We are unable to access/create data folder or data folder's
-      # permissions are insecure.
-      quit QuitFailure
+  trace "Data dir initialized", dir = $config.dataDir
+
+  if not(checkAndCreateDataDir((config.dataDir / "repo"))):
+    # We are unable to access/create data folder or data folder's
```

Comment:
Should these comments and those on lines 66, 67 be warn or error logs?

Reply:
Ah yeah, definitely want to print an error there and in the other place you pointed out.